src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -74,6 +76,11 @@ private static ArgumentParser getArgumentParser() {
                         + "If not provided, default file formats will be used.\n"
                         + "Please refer to userguide for more information.");
 
+        parser.addArgument("-start-server")
+                .action(storeTrue())
+                .help("Server will be started. "
+                        + "The dashboard will be displayed on the browser.");
```

Comment:
`Starts a server to display the browser`?
This argument provided is to do something, so probably should be more of an active verb.